src/schema/v2/artist/insights.ts

```diff
@@ -36,6 +37,13 @@ export const ArtistInsight = new GraphQLObjectType<any, ResolverContext>({
     },
     description: {
       type: GraphQLString,
+      args: {
+        ...markdown().args,
+      },
+      resolve: ({ description }, { format }) => {
+        if (!format) return formatMarkdownValue(description, "plain").trim()
```

Comment:
Would be better to set the `defaultValue` property in the args so that the resolver can be the same.

Reply:
yeah